src/edit.test.js

```diff
@@ -0,0 +1,19 @@
+/** ****************************************
+ * Project: js-todo-list
+ * File: edit.test.js
+ * Created: 3/9/23
+ * Author: Abdullah Al Mamun <mamun1214@gmail.com>
+ ****************************************** */
+import Apps from './modules/Apps';
+import Todo from './modules/Todo';
+
+describe('Edit function', () => {
+  const apps = new Apps();
+  const desc = 'This is changed item';
+  const $item = new Todo('Jest Test', false, 1);
+  apps.saveItems($item);
+  apps.update(desc, 1);
+  it('Edit Stored item', () => {
+    expect(apps.getItems()[0].description).toBe(desc);
+  });
+});
```

Comment:
Nice work creating the tests for the clear completed function. However, as part of the project requirements, you are to use mock for all functions that used localStorage, DOM or external APIs. Since the clear completed function uses the localStorage (the `saveItems` function on line 14 affects the local storage), kindly create a mock for the testing. :pray: